src/constants.tsx

```diff
@@ -1,197 +1,314 @@
 import { FIReasons, PlanActionCodes } from './configs/settings';
 import { FIReasonType } from './containers/forms/PlanForm/types';
 
-// strings
-export const MAP = 'map';
-export type MAP = typeof MAP;
-export const LOCATION = 'Location';
-export type LOCATION = typeof LOCATION;
-export const PROVINCE = 'Province';
-export type PROVINCE = typeof PROVINCE;
-export const HOME = 'Home';
-export type HOME = typeof HOME;
-export const FOCUS_INVESTIGATION = 'Focus Investigation';
-export type FOCUS_INVESTIGATION = typeof FOCUS_INVESTIGATION;
-export const CURRENT_FOCUS_INVESTIGATION = 'Current Focus Investigations';
-export type CURRENT_FOCUS_INVESTIGATION = typeof CURRENT_FOCUS_INVESTIGATION;
-export const COMPLETE_FOCUS_INVESTIGATION = 'Complete Focus Investigations';
-export type COMPLETE_FOCUS_INVESTIGATION = typeof COMPLETE_FOCUS_INVESTIGATION;
-export const INVESTIGATION = 'Investigation';
-export type INVESTIGATION = typeof INVESTIGATION;
-export const INTERVENTION = 'Intervention';
-export type INTERVENTION = typeof INTERVENTION;
-export const ACTIVE_FOCUS_INVESTIGATION = 'Active Focus Investigations';
-export type ACTIVE_FOCUS_INVESTIGATION = typeof ACTIVE_FOCUS_INVESTIGATION;
-export const FOCUS_AREA_INFO = 'Focus Area Information';
-export type FOCUS_AREA_INFO = typeof FOCUS_AREA_INFO;
-export const FOCUS_INVESTIGATIONS = 'Focus Investigations';
-export type FOCUS_INVESTIGATIONS = typeof FOCUS_INVESTIGATIONS;
+// magical strings
+export const JURISDICTION_ID = 'jurisdiction_id';
+export type JURISDICTION_ID = typeof JURISDICTION_ID;
+export const JURISDICTIONID = 'jurisdictionId';
+export type JURISDICTIONID = typeof JURISDICTIONID;
+export const PARENT_ID = 'parent_id';
+export type PARENT_ID = typeof JURISDICTION_ID;
+export const PARENTID = 'parentId';
+export type PARENTID = typeof JURISDICTIONID;
+export const CASE_TRIGGERED: FIReasonType = FIReasons[1];
+export const ROUTINE: FIReasonType = FIReasons[0];
+export const STRUCTURE_LAYER = 'structure-layer';
+export type STRUCTURE_LAYER = typeof STRUCTURE_LAYER;
+export const POLYGON = 'Polygon';
+export type POLYGON = typeof POLYGON;
+export const MULTI_POLYGON = 'MultiPolygon';
+export type MULTI_POLYGON = typeof MULTI_POLYGON;
+export const POINT = 'Point';
+export type POINT = typeof POINT;
+export const FEATURE = 'Feature';
+export type FEATURE = typeof FEATURE;
```

Comment:
@moshthepitt @p-netm - could we DRY this out by simply exporting all these strings... as strings? 

```
export const JURISDICTION_ID = 'jurisdiction_id';
export type JURISDICTION_ID = typeof JURISDICTION_ID;
```
becomes
```
export const JURISDICTION_ID = 'jurisdiction_id' as string;
```
🤔🤔🤔🤔🤔🤔

Reply:
@cKellyDesign yeah lets do this.  You do not even need the "as string" as Typescript will infer the type.